src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+                    System.out.println("____________________________________________________________\n"
+                            + " Noted. I've removed this task:\n" +
+                            "  " + tasks.get(index_for_delete - 1) + "\n" +
+                            "Now you have " + (i - 1) + " tasks in the list.\n"
+                            + "____________________________________________________________\n");
+                    i -= 1;
+                    tasks.remove(tasks.get(index_for_delete - 1));
+
+                }
+                String newString = "";
+                String filePath = "Duke.txt";
+
+                for (int m = 0; m < i; m += 1) {
+                    int index = m + 1;
+                    newString += (tasks.get(m) + "\n");
+                }
+
+
+                try {
+                    writeToFile("Duke.txt", newString);
+                } catch (IOException e) {
+                    System.out.println("Something went wrong\n"
+                            + "____________________________________________________________");
+                }
+            }
+
+            else if (line.toLowerCase().contains("todo") || line.toLowerCase().contains("deadline") || line.toLowerCase().contains("event")) {
+
+                boolean empty;
+                empty = false;
+
+                if (line.toLowerCase().contains("todo")) {
+                    String[] ToSplitTodo = line.split(" ");
+                    try {
+
+                        String TodoTask = line.toLowerCase().replaceAll("todo","");
+                        tasks.add(new Todo(TodoTask,ToSplitTodo.length));
+
+
+                    } catch (DukeException ex) {
+                        empty = true;
+                        System.out.println("____________________________________________________________\n"
+                                + "OOPS!!! The description of a todo cannot be empty.\n"
+                                + "____________________________________________________________\n");
+                    }
+
+                    //sample : deadline return book /by Sunday
+                } else if (line.toLowerCase().contains("deadline")) {
+                    String[] ToSplitDeadline = line.split("/");
+                    String DeadlineTask = ToSplitDeadline[0].toLowerCase().substring(9, ToSplitDeadline[0].length() - 1);
+                    tasks.add(new Deadline(DeadlineTask, ToSplitDeadline[1].substring(3, ToSplitDeadline[1].length())));
+
+                    //sample:event project meeting /from Mon 2pm /to 4pm
+                } else if (line.toLowerCase().contains("event")) {
+                    String[] ToSplitEvent = line.split("/");
+                    String EventTask = ToSplitEvent[0].toLowerCase().substring(6, ToSplitEvent[0].length());
+                    tasks.add(new Event(EventTask, ToSplitEvent[1].substring(5, ToSplitEvent[1].length()), ToSplitEvent[2].substring(3, ToSplitEvent[2].length())));
```

Comment:
Some of there lines are long >120 characters. If the line exceeds the limit, use line wrapping at appropriate places of the line. Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line. [See here](https://se-education.org/guides/conventions/java/basic.html#layout)